src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,249 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+
+    public static final int FIRST_INDEX = 1;
+    public static final int SECOND_INDEX = 2;
+    public static final int THIRD_INDEX = 3;
+
+    @Override
+    public Car create(Car car) {
+        String insertRequest = "INSERT INTO cars (manufacturer_id, model)\n "
+                               + "VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(insertRequest,
+                         Statement.RETURN_GENERATED_KEYS)) {
+            statement.setLong(FIRST_INDEX, car.getManufacturer().getId());
+            statement.setString(SECOND_INDEX, car.getModel());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(FIRST_INDEX, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't create "
+                                              + car + ". ", e);
+        }
+        setRelationsCarDrivers(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String getRequest = "SELECT c.id AS car_id, c.model, m.id AS manufacturer_id, \n"
```

Comment:
```suggestion
        String getRequest = "SELECT c.id AS car_id, c.model, m.id AS manufacturer_id, "
```
The same for the `\n`. You can replace them all on empty line by CTRL + R.

Reply:
> 

Cool, thank you for hint